book/Zapp-Pipes/protocols/login_and_payment_protocol.md

```diff
@@ -0,0 +1,47 @@
+# Login and Payment Protocol
+
+This document outlines the requirements for mapping the data to determine whether login and/or payment are required to gain access to a video item.
```

Comment:
```suggestion
This document outlines the requirements for mapping the data which determines whether login and/or payment are required to gain access to a video item.
```

Is it just for video items? Would it work for audio playables or articles or photo galleries?